lib/akamod/select-id.py

```diff
@@ -7,24 +7,28 @@
 from akara import logger
 from dplaingestion.selector import getprop, setprop, exists
 
-COUCH_ID_BUILDER = lambda src, lname: "--".join((src,lname))
-COUCH_REC_ID_BUILDER = lambda src, id_handle: COUCH_ID_BUILDER(src,id_handle.strip().replace(" ","__"))
+COUCH_ID_BUILDER = lambda src, lname: "--".join((src, lname))
+COUCH_REC_ID_BUILDER = lambda src, id_handle: COUCH_ID_BUILDER(src,
+                                                               CLEAN_ID(
+                                                                   id_handle))
+CLEAN_ID = lambda id_handle: id_handle.strip().replace(" ", "__")
+
```

Comment:
There was never a good reason for the "id builder" functions to be implemented as lambdas. They've been this way since [the initial commit in which they were added](https://github.com/dpla/ingestion/blob/5d7e78002/lib/akamod/select-id.py), without having been subjected to review, it appears. It would be nice to change these to functions as long as we're touching this code; one function, actually, named more appropriately to convey that it's constructing the CouchDB ID. Maybe another one that wraps `hashlib.md5()` and conveys that it's returning the DPLA ID for the given CouchDB ID.